src/index.ts

```diff
@@ -240,30 +242,51 @@ const notebooks: JupyterFrontEndPlugin<void> = {
 /**
  * A plugin that tracks notebook, console and file editors used for debugging.
  */
-const tracker: JupyterFrontEndPlugin<void> = {
+const tracker: JupyterFrontEndPlugin<TrackerHandler> = {
   id: '@jupyterlab/debugger:tracker',
   autoStart: true,
+  provides: IDebuggerEditorFinder,
   requires: [IDebugger, IEditorServices],
   optional: [INotebookTracker, IConsoleTracker, IEditorTracker],
   activate: (
     app: JupyterFrontEnd,
     debug: IDebugger,
     editorServices: IEditorServices,
+    editorFinder: IDebuggerEditorFinder
+  ) => {
+    return new TrackerHandler({
+      shell: app.shell,
+      editorServices,
+      debuggerService: debug,
+      editorFinder
+    });
+  }
+};
+
+/**
+ * A plugin that tracks editors, console and file editors used for debugging.
+ */
+const finder: JupyterFrontEndPlugin<EditorFinder> = {
+  id: '@jupyterlab/debugger:tracker',
```

Comment:
Should this be renamed to?

```suggestion
  id: '@jupyterlab/debugger:finder',
```